package.json

```diff
@@ -29,7 +29,7 @@
     "clean": "npm run clean --workspaces --if-present",
     "prettier": "npm run prettier --workspaces --if-present",
     "test:integration": "jest --config ./jest.config.ts --roots ./__tests__",
-    "test": "npm run test --workspaces --if-present",
+    "test": "NETWORK=Mainnet npm run test --workspaces --if-present",
```

Comment:
the nock fixtures were run against mainnet for some reason, will revert this when I update those if this change goes through